pkgs/applications/networking/instant-messengers/skypeforlinux/default.nix

```diff
@@ -68,7 +68,9 @@ in stdenv.mkDerivation {
 
   inherit src;
 
-  buildInputs = [ dpkg makeWrapper ];
+  nativeBuildInputs = [ wrapGAppsHook ];
+
+  buildInputs = [ dpkg gtk3 makeWrapper ];
```

Comment:
Alright, you do not need `gtk3` but `glib`’s setup hooks. Could you add it to `nativeBuildInputs` with a comment `for setup hook populating `GSETTINGS_SCHEMAS_PATH`?

Reply:
Also `makeWrapper` is not used.